packages/sample-apps/codegen/SampleAppJSI.h

```diff
@@ -0,0 +1,267 @@
+/**
+ * This code was generated by [react-native-codegen](https://www.npmjs.com/package/react-native-codegen).
+ *
+ * Do not edit this file as changes may cause incorrect behavior and will be lost
+ * once the code is regenerated.
+ *
+ * @generated by codegen project: GenerateModuleH.js
+ */
+
+#pragma once
+
+#include <ReactCommon/TurboModule.h>
+#include <react/bridging/Bridging.h>
+
+namespace facebook {
+namespace react {
+
+class JSI_EXPORT NativeMyJsiModuleCxxSpecJSI : public TurboModule {
```

Comment:
>NativeMyJsiModuleCxxSpecJSI [](http://example.com/codeflow?start=17&length=27)

What is the difference between this NativeMyJsiModuleCxxSpecJSI and NativeMyModuleCxxSpecJSI below? They are both using JSI TurboModules. They look like code duplication to me. The same is in the .cpp file.

Reply:
Ah, we put generated specs for all modules in a project into one file! It makes sense.